app/main.py

```diff
@@ -1 +1,38 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = 1) -> None:
```

Comment:
The default value for `is_hungry` should be a boolean (`True` or `False`) instead of an integer (`1`). This will ensure consistency with the expected data type for boolean values.